J/Julia/build_tarballs.jl

```diff
@@ -3,99 +3,68 @@
-    ;;
-    arm-*)
-        BUILD_FLAGS+=("MARCH=armv7-a" "JULIA_CPU_TARGET=armv7-a;armv7-a,neon;armv7-a,neon,vfp4")
-    ;;
-    ppc64le-*)
-        BUILD_FLAGS+=("JULIA_CPU_TARGET=pwr8")
-    ;;
-    aarch64-*)
-        BUILD_FLAGS+=("MARCH=armv8-a" "JULIA_CPU_TARGET=generic")
-    ;;
-esac
-
-# If we're compiling for Windows, then set XC_HOST
-if [[ ${target} == *mingw* ]]; then
-    BUILD_FLAGS+=("XC_HOST=${target}")
+if [[ ${target} == x86_64-*mingw* ]]; then
+	apk add p7zip
+	7z e /storage/downloads/*-julia-1.3.1-win64.exe
+	cd ${prefix}
+	7z e ${WORKRSPACE}/srcdir/julia-installer.exe
+elif [[ ${target} == i686-*mingw* ]]; then
+	apk add p7zip
+	7z e /storage/downloads/*-julia-1.3.1-win32.exe
+	cd ${prefix}
+	7z e ${WORKRSPACE}/srcdir/julia-installer.exe
+elif [[ ${target} == arm-linux-gnueabihf ]]; then
+	cd ${prefix}
+	tar xzf /storage/downloads/*julia-1.3.1-linux-aarch64.tar.gz
+elif [[ ${target} == x86_64-linux-gnu ]]; then
+	cd ${prefix}
+	tar xzf /storage/downloads/*julia-1.3.1-linux-x86_64.tar.gz
+elif [[ ${target} == i686-linux-gnu ]]; then
+	cd ${prefix}
+	tar xzf /storage/downloads/*julia-1.3.1-linux-i686.tar.gz
+elif [[ ${target} == x86_64-apple-darwin* ]]; then
+	cd ${prefix}
+	apk add p7zip
+	7z x /storage/downloads/*julia-1.3.1-mac64.dmg
 else
-    export LDFLAGS="$LDFLAGS -Wl,-rpath,${prefix}/lib"
+	echo "ERROR: Unsupported platform ${target}" >&2
+        exit 1
 fi
-
-# Make use of many prebuilt things
-#for proj in BLAS LLVM PCRE MBEDTLS LIBSSH2 CURL LIBGIT2 GMP MPFR; do
-#    BUILD_FLAGS+=(USE_SYSTEM_${proj}=1)
-#done
-
-#if [[ ${nbits} == 64 ]]; then
-#    BUILD_FLAGS+=(LIBBLASNAME=libopenblas LIBBLAS=-lopenblas64_)
-#else
-#    BUILD_FLAGS+=(LIBBLASNAME=libopenblas LIBBLAS=-lopenblas)
-#fi
-
-#BUILD_FLAGS+=(LDFLAGS="$LDFLAGS")
-BUILD_FLAGS+=(LIBSSH2_ENABLE_TESTS=0)
-
-make ${BUILD_FLAGS[@]} -j${nproc} -C deps install-mbedtls
-make ${BUILD_FLAGS[@]} -j${nproc} -C deps install-libgit2
-make ${BUILD_FLAGS[@]} -j${nproc}
-make ${BUILD_FLAGS[@]} install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
-    Linux(:x86_64; libc=:glibc, compiler_abi=CompilerABI(:gcc7)),
-    Linux(:i686; libc=:glibc, compiler_abi=CompilerABI(:gcc7)),
-    Windows(:x86_64; compiler_abi=CompilerABI(:gcc7)),
-    Windows(:i686; compiler_abi=CompilerABI(:gcc7)),
+    Linux(:x86_64; libc=:glibc),
+    Linux(:i686; libc=:glibc),
+    Windows(:x86_64;),
+    Windows(:i686;),
+    MacOS(:x86_64)
 ]
 
 # The products that we will ensure are always built
-products(prefix) = Product[
-    ExecutableProduct(prefix, "julia", :julia),
-    LibraryProduct(prefix, "libjulia", :libjulia),
+products = Product[
+    ExecutableProduct("julia", :julia),
+    LibraryProduct("libjulia", :libjulia),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
```

Comment:
Doesn't this still need binary dependencies?

Reply:
I don't think so. But maybe I am forgetting something.